### PR TITLE
Update slack_bot.js

### PR DESCRIPTION
**Pull request by techforevil**
changed name to Michael Bot-Spall

still needs new profile pic

### CODE REVIEW

File: slack_bot.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ controller.hears(['^announce (.*)'],
               form: {
                 channel: channel.id,
                 token: process.env.apitoken,
-                username: "thegovernor",
+                username: "Michael Bot-Spall",
```
Comment (surminus) [inline slack_bot.js]: does this username parameter work with spaces?

**Comment by edwardhorsford** [comment]
I approve of this change.

File: slack_bot.js (hunk 1 above)
Comment (techforevil) [reply slack_bot.js]: very good question 

i do not know

**Comment by adammaddison** [comment]
How could anyone not approve of this change?

**Comment by bruntonspall** [comment]
That's not how the name is configured for most messages.

The bot replies to the thread or responds for almost all messages, but this bit of code is there so it can post a brand new message, so it uses the REST API.

So this would only change the name for @announce messages rather than for the entire bot.   The bot's name elsewhere is configured using the slack admin page (and I don't think can be configured in code)

**Pull request merged**